cockpit-frontend/src/components/ScrumMasterForm/ScrumMasterForm.jsx

```diff
@@ -0,0 +1,277 @@
+import React, { useState } from 'react';
+import Modal from '@material-ui/core/Modal';
+import { useSelector, useDispatch } from 'react-redux';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import FormLabel from '@material-ui/core/FormLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import Button from '@material-ui/core/Button';
+import CancelIcon from '@material-ui/icons/Cancel';
+import IconButton from '@material-ui/core/IconButton';
+import { showSMFormState, close } from '../Header/HeaderSlice';
+import {
+  SMFormState,
+  setMvp,
+  setJiraId,
+  setEntity,
+  setCycle,
+  setFormIsValid,
+  setImageUrl,
+  imageUrlState,
+  mvpState,
+  jiraIdState,
+  entityState,
+  cycleState,
+} from './ScrumMasterFormSlice';
+import '../../App.scss';
+import useStyles from './styles';
+import MvpService from '../../services/service';
+import { fetchAllMvps } from '../../redux/ormSlice';
+
+export default function ScrumMasterForm() {
+  const classes = useStyles();
+
+  let url = '';
+  let btnClass = [classes.bubbleBox, 'bubble'];
+  btnClass = btnClass.join(' ');
+
+  const [isOpened, setIsOpened] = useState(false);
+  const imageUrl = useSelector(imageUrlState);
+  const mvpName = useSelector(mvpState);
+  const jira = useSelector(jiraIdState);
+  const selectedEntity = useSelector(entityState);
+  const cycle = useSelector(cycleState);
+  const isFormValid = useSelector(SMFormState);
+  const open = useSelector(showSMFormState);
+  const dispatch = useDispatch();
+
+  function openDialog() {
+    setIsOpened(true);
+  }
+  function closeDialog() {
+    setIsOpened(false);
+  }
+  function confirmImageUrl() {
+    if (url !== '') {
+      dispatch(setImageUrl(url));
+      dispatch(setFormIsValid());
+      url = '';
+      closeDialog();
+    }
+  }
+  function removeImageUrl() {
+    dispatch(setImageUrl(null));
+  }
+  function handleChange(event) {
+    url = event.target.value;
+  }
+  async function submit(e) {
```

Comment:
place const object in a separated script, we can create a folder like src/common/mockData for example